Description: Use python3 for dfuse-pack.py
 Upstream has /usr/bin/python in the shebang since most non-Debian
 systems uses Python 3 as "python". Although the script will work
 with either Python 2 or 3, the shebang must match our dependency
 in debian/control.
Author: Tormod Volden <debian.tormod@gmail.com>
Forwarded: not-needed
Last-Update: 2022-01-21

--- dfu-util-0.11.orig/dfuse-pack.py
+++ dfu-util-0.11/dfuse-pack.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 # Written by Antonio Galea - 2010/11/18
 # Distributed under Gnu LGPL 3.0
